src/conditions/BUILD

```diff
@@ -10,6 +10,12 @@ filegroup(
     visibility = ["//src:__pkg__"],
 )
 
+config_setting(
+    name = "linux_ppc",
```

Comment:
Should`ppc` should imply `Linux`? I see there's already [precedent](https://github.com/bazelbuild/bazel/pull/5525/files#diff-5aeb8330f796a1fbec8885e37f3ca5b9R56) for `aarch64` so I guess it's not a huge deal.

Reply:
yah that's just a convention i've seen and stuck with. but you're right, for now bazel doesn't run on aix or i :)